app/src/main/java/no/usn/mob3000/App.kt

```diff
@@ -157,9 +158,16 @@ fun App(
             composable(route = Destination.GROUPS_CREATE.name) { CreateGroupScreen() }
             composable(route = Destination.PLAY.name) { PlayScreen() }
             composable(route = Destination.HISTORY.name) { HistoryScreen() }
-            composable(route = Destination.PROFILE.name) { ProfileScreen() }
+            composable(route = Destination.PROFILE.name) { /* ProfileScreen() */ LoginScreen(
```

Comment:
This is fine during testing, but is not acceptable for merging into master. The profile route must go to the profile screen. Remove the incorrect call to `LoginScreen` and uncomment `ProfileScreen`.